.github/workflows/global-api-develop.yml

```diff
@@ -11,6 +11,14 @@ on:
       - k8s/cc-global-api-migrate.yml
       - .github/workflows/global-api-develop.yml
     branches: ["develop"]
+  pull_request:
+    paths:
+      - global-api/**
+      - k8s/cc-global-api-deploy.yml
+      - k8s/cc-global-api.yml
+      - k8s/cc-global-api-migrate.yml
+      - .github/workflows/global-api-develop.yml
+    branches: ["develop"]
```

Comment:
Don't need to use `develop` here, as it applies for pull requests into any branch

Reply:
You can just remove the branches line here